src/api/Base.js

```diff
@@ -393,6 +396,27 @@ class Base {
             this.errorHandler(error);
         }
     }
+
+    /**
+     * Fill user properties that are null in an object
+     *
+     * @param {Object} obj - some object
+     * @return {Object} new object with user placeholder
+     */
+    fillUserPlaceholder(obj: Object): Object {
+        const newObj = { ...obj };
+
+        USER_FIELDS.forEach((field: string) => {
+            if (
+                Object.prototype.hasOwnProperty.call(newObj, field) &&
```

Comment:
should this be `newObj.hasOwnProperty(field)`?

Reply:
I had it as that originally... then eslint told me I was wrong, but based on the eslint rules now it appears that calling `hasOwnProperty` on newObj is acceptable again!